react/features/overlay/components/PageReloadOverlay.js

```diff
@@ -0,0 +1,292 @@
+/* global APP, AJS */
+import React, { Component } from 'react';
+import { randomInt } from '../../base/util/randomUtil';
+import AbstractReloadableOverlay from './AbstractReloadableOverlay';
+const logger = require('jitsi-meet-logger').getLogger(__filename);
```

Comment:
It will work but it's going to look like this:
```
import {getLogger} from "jitsi-meet-logger";
const logger = getLogger(__filename);
```
Which is longer. 

Also currently this the way how the logger is used by the files from the react folder.